scripts/convert_hf_checkpoint.py

```diff
@@ -82,17 +82,21 @@ def copy_weights_falcon(
     }
     # the original model definition is different for each size
     if "7b" in model_name:
-        weight_map.update({
-            "transformer.h.{}.input_layernorm.bias": "transformer.h.{}.norm_1.bias",
-            "transformer.h.{}.input_layernorm.weight": "transformer.h.{}.norm_1.weight",
-        })
+        weight_map.update(
```

Comment:
ruff and/or black behave differently in Studio and on my local machine. I used the latest packages with the same CLI code. Weird.
@carmocca could you before merging, when the PR is ready, to run formatter one more time?

Reply:
Just pushed a commit to main after upgrading black

Reply:
Wonderful, they've changed their mind about the way to format files 😆